### PR TITLE
add_na_to_rationale: Add n/a to accepted components rationale

### LINKED ISSUE #892: Add "n/a" to accepted components rationale
### Summary

This issue stems from the following neurostars post: https://neurostars.org/t/tedana-v0-0-12-cant-read-metrics-tsv-file/23810

### Next Steps

- Add na_rep='n/a' to to_csv calls.

### PR DESCRIPTION
**Pull request by karimox**
save_tsv, OutputGenerator

Closes #892
Add "n/a" to accepted components rationale 

Changes proposed in this pull request:

- in tedana/io.py, I modified the OutputGenerator.save_tsv method:
    rows with accepted classifications get the value 'n/a' in rationale  
    'n/a' will be read as NaN by pandas when loading the file

### CODE REVIEW

**Comment by jbteves** [comment]
Looks like CircleCI is currently offline for us... I'll see if I can figure it out.

**Comment by jbteves** [comment]
Thanks for your contribution, @karimox ! Unfortunately our test suite has decided that today is a good day to go on holiday early, so it may take a while to figure out how to get that back up.

**Comment by jbteves** [comment]
@karimox @eurunuela hilariously mentioning this in the new PR has kicked off the tests.

File: tedana/io.py [hunk 1]
```diff
@@ -266,7 +266,9 @@ def save_tsv(self, data, name):
         data_type = type(data)
         if not isinstance(data, pd.DataFrame):
             raise TypeError(f"data must be pd.Data, not type {data_type}.")
-        data.to_csv(name, sep="\t", line_terminator="\n", na_rep="n/a", index=False)
+        if "rationale" in data.columns and "classification" in data.columns:
+            data.loc[data['classification'] == 'accepted', 'rationale'] = 'n/a'
```
Comment (tsalo) [inline tedana/io.py:270]: black will complain about the single quotes:
```suggestion
            data.loc[data["classification"] == "accepted", "rationale"] = "n/a"
```

Comment (tsalo) [inline tedana/io.py:270]: What about something simpler, like:

```suggestion
        data.replace("", "n/a")
```

**Comment by handwerkerd** [comment]
Need to check if this is still an issue after merging #756. `classification_tags` has replaced `rationale` so, if we're merging something like this it would need to be lined up with the current code. It is possible for `classification_tags` to be empty, but that won't happen in any of the trees that are distributed with ``tedana`` since every component will end up classified as `accepted` or `rejected` and will have a text string in `classification_tags`

**Comment by handwerkerd** [comment]
Since this issue should no longer be possible if `classification_tags` are used properly, I'm closing this PR.

**Pull request closed without merging**